pkg/gcv/configs/config.go

```diff
@@ -460,8 +461,49 @@ func convertLegacyConstraintTemplate(u *unstructured.Unstructured, regoLib []str
 	return nil
 }
 
+var starRegex = regexp.MustCompilePOSIX(`/\*$`)
```

Comment:
This will only match the end of the string, should we also match `organizations/*/folder/mine`?

Reply:
added